charts/osdfir-infrastructure/charts/openrelik/values.yaml

```diff
@@ -158,41 +170,90 @@ metrics:
+  image: ghcr.io/openrelik/openrelik-worker-extraction:0.4.0
+  command: "celery --app=src.app worker --task-events --concurrency=1 --loglevel=INFO -Q openrelik-worker-extraction"
+  privileged: false
+  mountHostDevices: false
+  replicas: 1
   env: {}
   resources: {}
+  nodeSelector: {}
 - name: openrelik-worker-analyzer-config
-  image: ghcr.io/openrelik/openrelik-worker-analyzer-config:2024.11.27
+  image: ghcr.io/openrelik/openrelik-worker-analyzer-config:0.2.0
+  command: "celery --app=src.app worker --task-events --concurrency=1 --loglevel=INFO -Q openrelik-worker-analyzer-config"
+  privileged: false
+  mountHostDevices: false
+  replicas: 1
   env: {}
   resources: {}
-  command: "celery --app=src.app worker --task-events --concurrency=4 --loglevel=INFO -Q openrelik-worker-analyzer-config"
+  nodeSelector: {}
 - name: openrelik-worker-hayabusa
-  image: ghcr.io/openrelik/openrelik-worker-hayabusa:2024.11.27
-  command: "celery --app=src.app worker --task-events --concurrency=4 --loglevel=INFO -Q openrelik-worker-hayabusa"
+  image: ghcr.io/openrelik/openrelik-worker-hayabusa:0.3.0
+  command: "celery --app=src.app worker --task-events --concurrency=1 --loglevel=INFO -Q openrelik-worker-hayabusa"
+  privileged: false
+  mountHostDevices: false
+  replicas: 1
   env: {}
   resources: {}
+  nodeSelector: {}
+- name: openrelik-worker-yara
+  image: ghcr.io/openrelik/openrelik-worker-yara:latest
+  command: "celery --app=src.app worker --task-events --concurrency=1 --loglevel=INFO -Q openrelik-worker-yara"
+  privileged: false
+  mountHostDevices: false
+  replicas: 1
+  env: {}
 ## When deployed via the OSDFIR Infrastructure chart, the Timesketch Worker automatically receives its required environment 
 ## variables through Helm.
 ##
 - name: openrelik-worker-timesketch
-  image: ghcr.io/openrelik/openrelik-worker-timesketch:2024.11.27
+  image: ghcr.io/openrelik/openrelik-worker-timesketch:0.3.0
   command: "celery --app=src.app worker --task-events --concurrency=1 --loglevel=INFO -Q openrelik-worker-timesketch"
+  privileged: false
+  mountHostDevices: false
+  replicas: 1
   env: {}
   resources: {}
+  nodeSelector: {}
 ## @section OpenRelik Configuration Parameters
 ##
 config:
+  ## @param config.existingSecret Use an existing secret as the OpenRelik secret.
+  ## Secret name must follow the naming convention of {{ .Release.Name }}-openrelik-secret
+  ##
+  existingSecret: false
+  ## OpenRelik nbd init container configuration for OpenRelik containers that require the nbd module.
+  ## Only enable this container when you need to load the nbd kernel module to the underlying node.
+  ##
+  initWorkerNbd:
+    ## @param config.initWorkerNbd.enabled Whether to enable the nbd init container
+    ##
+    enabled: false
+    ## @param config.initWorkerNbd.image Container image with modprobe installed necessary
+    ## for loading the nbd module
+    ##
+    image: "us-central1-docker.pkg.dev/osdfir-registry/openrelik/openrelik-nbd:0.1.0"
```

Comment:
This is the image name for the init container that will setup the nbd module. `us-central1-docker.pkg.dev/osdfir-registry/openrelik/openrelik-nbd:0.1.0` 

I will add that to my todo list to build that container. Yes?